### PR TITLE
UNPLUGGED-115 | Remove internal search tab from plugin configuration

### PR DESCRIPTION
**Pull request by frobs**
Required by #115

### CODE REVIEW

**Review by albertodoofinder — APPROVED** [review]

File: views/templates/admin/configure.tpl [hunk 1]
```diff
@@ -36,7 +36,6 @@
     {else}
     <li class="active"><a href="#data_feed_tab" role="tab" data-toggle="tab">{l s='Data Feed' mod='doofinder'}</a></li>
     {if !$dfEnabledV9 && !$is_new_shop}
-        <li><a href="#internal_search_tab" role="tab" data-toggle="tab">{l s='Internal Search' mod='doofinder'}</a></li>
         <li><a href="#custom_css_tab" role="tab" data-toggle="tab">{l s='Custom CSS' mod='doofinder'}</a></li>
```
Comment (ivan-doofinder) [inline views/templates/admin/configure.tpl]: Also delete this tab. it's useless (custom_css_tab)

**Review by ivan-doofinder — CHANGES_REQUESTED** [review]

File: views/templates/admin/configure.tpl (hunk 1 above)
Comment (frobs) [reply views/templates/admin/configure.tpl]: Removed

Comment (mdogo) [reply views/templates/admin/configure.tpl]: > Also delete this tab. it's useless (custom_css_tab)

Are you sure? Isn't it used by the Layer v7?

Comment (ivan-doofinder) [reply views/templates/admin/configure.tpl]: All v7 related settings have been removed. If a user with v7 installs the latest version of the plugin, they will still be able to use the v7 layer, but will not have access to any related settings. The settings you have when you update are kept.

**Comment by ivan-doofinder** [comment]
Remember to update the plugin version before the release creation

**Review by ivan-doofinder — APPROVED** [review]

File: views/templates/admin/configure.tpl (hunk 1 above)
Comment (frobs) [reply views/templates/admin/configure.tpl]: @ivan-doofinder  what do we do? Do we remove the custom css tab too or do we keep it?

Comment (ivan-doofinder) [reply views/templates/admin/configure.tpl]: The tab currently doesn't work, so it doesn't make sense for it to be there.

**Review by mdogo — APPROVED** [review]

**Pull request merged**